x/interchainqueries/types/tx.go

```diff
@@ -91,6 +91,22 @@ func (msg MsgRegisterInterchainQuery) ValidateBasic() error {
 		return sdkerrors.Wrap(ErrInvalidQueryType, "invalid query type")
 	}
 
+	if InterchainQueryType(msg.QueryType).IsKV() {
+		if len(msg.Keys) == 0 {
+			return sdkerrors.Wrap(ErrEmptyKeys, "keys cannot be empty")
+		}
+
+		// check msg.Keys has path
+		for _, key := range msg.Keys {
```

Comment:
There is a couple of paragraphs in the audit report about unbounded loops. What do you think about this one?

Reply:
true; did we introduce a cap for kv keys? if not, we should, and we should also check that cap during query creation

Reply:
so should I add one more env var to store this param or lets say set it like constant = 128 or something?

Reply:
I personally think that constant is fine. But maybe there are some other thoughts

Reply:
just a thought that we have pile of shit from constants already, lol

Reply:
If we are going to stick with governance params then we can't check `.Keys` length in `ValidateBasic()`. That's why I like constant solution more

Reply:
please just make a constant, it is the easiest one to implement and change in future, if desired,